examples/ecs/system_closure.rs

```diff
@@ -0,0 +1,35 @@
+use bevy::{log::LogPlugin, prelude::*};
+
+fn main() {
+    // create a non-capturing closure.
+    let non_capturing_closure = || {
+        info!("Hello from a non-capturing closure!");
+    };
+
+    // create a capturing closure, with a Local value.
+    let capturing_closure = |value: String| {
```

Comment:
this is confusing as this is not a capturing closure

Reply:
I think I miss understood what's a capturing closure, I'll change that when I come back after school.